src/formatting.rs

```diff
@@ -73,6 +73,20 @@
 //! `format` or `f`        | [chrono docs](https://docs.rs/chrono/0.3.0/chrono/format/strftime/index.html#specifiers) for all options. | `'%a %d/%m %R'`
 //! `locale` or `l`        | Locale to apply when formatting the time                                                                  | System locale
 //!
+//!
+//! ## `duration`/`dur` - Format durations
+//!
+//! Argument                     | Description                                                                                      |Default value
+//! -----------------------------|--------------------------------------------------------------------------------------------------|---------------------------------------------
+//! `hms`                        | Should the format be hours:minutes:seconds.milliseconds                                          | `false`
+//! `max_unit`                   | The largest unit to display the duration with                                                    | hms ? `h` : `y`
+//! `min_unit`                   | The smallest unit to display the duration with                                                   | `s`
```

Comment:
I think it would be very useful to have a list of all possible units. For example below the table, with a note "see below for the list of all possible units".